internal/notification/data/README.md

```diff
@@ -0,0 +1,45 @@
+# Customer Notifications
+
+## Overview
+
+Issues related to OSD addons may require that a notification be sent to the
+cluster owner in the event that a SRE team supporting the addon is required
+to perform some manual action.
+
+This directory hosts configurations for such notifications which may be sent
+using the `ocm addons notify` command.
+
+## Adding New Teams
```

Comment:
I lack context on this feature, but was there ever a debate of organizing notifications per product instead of per team? It feels to me that CSSRE and MTSRE might want to share some notification templates.

Reply:
This is loosely based off of https://github.com/openshift/managed-notifications which kind of does what you suggest, but is very **SREP** specific. I think any shared notifications can just go in a `common` folder. Don't remember exactly, but I'm pretty sure I wanted team directories in case we wanted to add an `OWNERS` file or something to give other teams more control.

Reply:
I like the `common` directory idea